chain-registry/src/utils.rs

```diff
@@ -10,48 +10,49 @@ pub const REF: &str = "ffdfbff3a21c7d2dbaac6f1e4730f47063f4bd72";
 
 pub const TEST_CHAINS: &[&str] = &["cosmoshub", "juno", "evmos", "osmosis", "regen"];
 
-pub trait FileName {
+#[async_trait]
```

Comment:
We should document this trait and its methods. A suggestion:
```suggestion
/// `Fetchable` represents the basic expectations for external data or resources that
/// can be fetched.
#[async_trait]
```